blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,135 @@
+    channelLogo,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    itemDescription,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      ...(channelTitle && {title: channelTitle}),
+      ...(audioAvailable && {Audio: audioAvailable}),
+      link: `${domain}`,
+      ...(channelDescription && {description: channelDescription }),
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+    
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let body, category
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        return {
+          title: jmespath.search(s, itemTitle) || '',
+          link: url,
+          guid: s._id,
+          pubDate: s[pubDate],
+          ...(itemCategory &&
+            (category = jmespath.search(s, itemCategory)) &&
+            category && { category: category }),
+          ...(includeContent !== 0 &&
+            (body = cheerio
+              .load(
+                rssBuildContent.parse(
+                  s.content_elements || [],
+                  includeContent,
+                  domain,
+                  resizerKey,
+                  resizerURL,
+                  resizerWidth,
+                  resizerHeight,
+                ) || '',
+              )
+              .text('bodyContent')) &&
+            body && {
+              description: body,
+            }),
+        }
+      }),
+    },
+  },
+})
+
+export function Rss({ globalContent, customFields, arcSite }) {
+  const {
+    resizerURL = '',
+    feedDomainURL = '',
+    feedTitle = '',
+    feedLanguage = '',
+  } = getProperties(arcSite)
+
+  const { width = 0, height = 0 } = customFields.resizerKVP || {}
+
+  const rssBuildContent = new BuildContent();
+
+  // can't return null for xml return type, must return valid xml template
+  return rssTemplate(get(globalContent, 'content_elements', []), {
+    ...customFields,
+    resizerURL,
+    resizerWidth: width,
+    resizerHeight: height,
+    domain: feedDomainURL,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  })
+}
+
+Rss.propTypes = {
+  customFields: PropTypes.shape({
+    audioAvailable: PropTypes.string.tag({
+      label: 'Audio',
+      group: 'Audio',
+      description: 'description',
+      defaultValue: 'content_elements[?type==audio].streams[0].url', 
```

Comment:
Sorry the formatting got messed up when I pasted this into jira.  The word audio needs to be in backticks.  Also add the '|[0]' otherwise it returns an array.

'content_elements[?type==`audio`].streams[].url|[0]'